trin-core/src/jsonrpc/endpoints.rs

```diff
@@ -34,6 +34,8 @@ pub enum HistoryEndpoint {
     RecursiveFindContent,
     Store,
     RoutingTableInfo,
+    // This endpoint is not History network specific
+    PaginateLocalContentKeys,
```

Comment:
The problem with making this a `PortalEndpoint` is that `PortalEndpoint`s get forwarded onto the `eth_handler` rather than directly to a subnetwork. This is a redundant layer of abstraction that is unnecessary for the function of this endpoint. Since, the `eth_handler` (which doesn't have direct access to storage) will just simply forward the request on to the history subnet. Rather than introduce that redundant forwarding, I thought it would be simpler to just process this endpoint under the guise of a `HistoryEndpoint` - even though it's technically not. 

I don't have a strong opinion on this, if you guys think it's worthwhile to introduce that extra forwarding, in order to maintain the integrity of the naming scheme, I can make the change.

Reply:
> I don't have a strong opinion on this, if you guys think it's worthwhile to introduce that extra forwarding, in order to maintain the integrity of the naming scheme, I can make the change.

This makes sense, we are duplicating many of the endpoints already in `StateEndpoint` and `HistoryEndpoint`. 
I think the simplest solution is to leave it as it is, if we want something more robust, we need to refactor some of the `JsonRpcHandler` code and the endpoints which are out of the scope of this PR.